Sources/Sentry/SentrySessionReplayIntegration.m

```diff
@@ -102,9 +102,12 @@ - (void)setupWith:(SentryReplayOptions *)replayOptions
 
     id<SentryViewRenderer> viewRenderer;
     if (enableViewRendererV2) {
+        SENTRY_LOG_DEBUG(@"[Session Replay] Setting up view renderer v2, fast view rendering: %@",
```

Comment:
`m`: Maybe we should define a constant for `[Session Replay]` or maybe even extra macros to ensure we're using the `[Session Replay] ` for filtering. It doesn't have to be in this PR. My main concern is that it's very easy to not follow the patten of using the `[Session Replay]` prefix, and I would like to find a way to ensure this pattern. We could also have a linter checking that log messages start with this pattern if the class is a SR class.

Reply:
I would propose we go one step further and introduce "categories" for logs, similar to OSLog [1]. Maybe we can do this easily be introducing a parameter for the macro instead of using a  string-based approach. I know this will introduce other issues down the road, so yes different PR.

My intention with the pattern was not introducing a standardized approach, I just needed a way to filter logs for session replay.

[1] https://www.avanderlee.com/debugging/oslog-unified-logging/

Reply:
Let's solve this in another PR.